### PR TITLE
Add changes from deployment branch

### PR DESCRIPTION
**Pull request by jenmerritt**
### Description

- This branch was created off `v200.develop` - the branch containing all the modularized samples.
- The daily sample viewer builds have been building off this `samples-deployment-test` branch to create the new installers so it has been thoroughly tested from that side. 
- The key changes in this branch to merge into `v200.develop` are:
   - includes all the version updates to gradle, JavaFX and other dependencies. This involves the `build.gradle`, `module-info` and `gradle-wrapper.properties` file for each sample (213 * 3 = 639 changed files)
   - Additional changes made to:
      - Readme.md
      - Image used in readme
      - WorkingWithGit.md
      - Root build.gradle
      - Root settings.gradle

As a confidence check, that's 639 changed sample files, plus the above 5, which makes 644 changed files total, per the PR.

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by jenmerritt** [comment]
@01smito01 please could you review?

File: README.md [hunk 1]
```diff
@@ -14,82 +14,83 @@
 
 ![Gradle build](https://github.com/Esri/arcgis-runtime-samples-java/workflows/Java%20CI%20with%20Gradle/badge.svg)
 
-Welcome to the home of the ArcGIS Maps SDK for Java samples! The ArcGIS Maps SDK for Java is used to build mapping, location, and GIS applications for desktop applications running on Windows, Mac, or Linux. This repo contains a set of sample projects demonstrating how to accomplish various mapping and geospatial tasks with the ArcGIS Maps SDK for Java.
+Welcome to the ArcGIS Maps SDK for Java Samples.
```
Comment (01smito01) [inline README.md]: Maybe it's just bc this is different, but this line looks a bit strange to me. Maybe "Welcome to the ArcGIS Maps SDK for Java Samples repo." instead?

File: README.md [hunk 2]
```diff
@@ -14,82 +14,83 @@
...
 
-![Choose Camera Controller sample preview](https://user-images.githubusercontent.com/36415565/185649571-7d6feb6f-f0c2-42cb-9139-9d4541dd1da8.png)
+The ArcGIS Maps SDK for Java is used to build mapping, location, and GIS applications for desktop applications running on Windows, Mac, or Linux. This repo contains a set of sample projects demonstrating how to accomplish various mapping and geospatial tasks with the ArcGIS Maps SDK for Java.
 
-Browse the category directories to explore the samples. Each sample is an individual [Gradle](https://docs.gradle.org/current/userguide/userguide.html) project that can be run standalone. The Gradle buildscripts have tasks for running the application, building a jar, and distributing the app as a zip.
+### Sections
+* [Sample Viewer](#sampler-viewer)
```
Comment (01smito01) [inline README.md]: Fix spelling on `#sample-viewer`

File: README.md [hunk 3]
```diff
@@ -14,82 +14,83 @@
...
+* [Running the samples locally via this project](#running-the-samples-locally-via-this-project)
+* [Setup an API Key](#setup-an-api-key)
+* [Offline sample data](#offline-sample-data)
+* [System Requirements for ArcGIS Maps SDK for Java](#system-requirements-for-arcgis-maps-sdk-for-java)
```
Comment (01smito01) [inline README.md]: Is "System Requirements" a proper noun? Based on how the other titles are done shouldn't the r in requirements be lower case?

File: README.md [hunk 4]
```diff
@@ -14,82 +14,83 @@
...
+### Setup an API key
+Accessing Esri location services, including basemaps, routing, and geocoding, requires authentication using either an ArcGIS identity or an API Key:
+1. ArcGIS identity: An ArcGIS named user account that is a member of an organization in ArcGIS Online or ArcGIS Enterprise.
+2. API key: A permanent key that gives your application access to Esri location services. Visit your [ArcGIS Developers Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.
 
 ### Offline sample data
 Some samples require offline data. A `samples-data` directory will automatically download to the project root when the Gradle project is configured/imported.
 
-## Requirements
+### System Requirements for ArcGIS Maps SDK for Java
```
Comment (01smito01) [inline README.md]: Same q as before wrt capitalisation

**Review by 01smito01 — APPROVED** [review]
Works locally w/o issue, only minor suggestions regarding `README.md`. I don't see the promised changes to `settings.gradle` though (and yet your maths works. odd.)

File: README.md (hunk 2 above)
Comment (jenmerritt) [reply README.md]: Ooh good catch!

File: README.md (hunk 3 above)
Comment (jenmerritt) [reply README.md]: Yep since the website has lower case I'll follow 👍

**Comment by jenmerritt** [comment]
I'm going to merge this after one review, since it's feature branch to feature branch. Will get 2 reviewers on the next PR...

**Comment by jenmerritt** [comment]
Scratch that. I'm going to instead target this at `v.next` after it is updated from `v200.develop`, while the developer site is pointed at `v200.develop`. To keep the version updates into the next release.

**Comment by jenmerritt** [comment]
Closing to avoid confusion and will reopen and retarget after the next PR is done.

**Pull request merged**